src/modules/web-actions/index.js

```diff
@@ -0,0 +1,51 @@
+import {NxusModule} from 'nxus-core'
+import {templater} from 'nxus-templater'
+import _ from 'underscore'
+
+class WebActions extends NxusModule {
+  constructor(opts) {
+    super(opts)
+    this._actions = {}
+    templater.templateDir(__dirname+"/templates/*.ejs")
+    templater.on('renderContext.actions-buttons', ::this._actionsContext)
+    templater.on('renderContext.actions-icons', ::this._actionsContext)
+    
+  }
+  
+  /*
+   * Register an action for a template
+   * @param {String} template
+   * @param {String} label
+   * @param {String} link
+   * @param {Object} options (icon, css, group)
+   */
+  add(template, label, link, options) {
```

Comment:
Hmm, I like it. Definitely for letting you build little forms/posts rather than just links. A name of a template partial in options:
```
templater.template('my-action-partial.ejs')
actions.add('admin-whatever-list', "Select", "/select", {template: 'my-action-partial'}
```
And can register the just-one-button/icon as the default values for action.partial.

Question of whether the "actions-buttons" vs "actions-icons" full/minimal display choice is a key feature here, should there be both `template` and `templateMinimal` options?